MediaGalleryUi/Console/Command/IndexAssets.php

```diff
@@ -44,6 +56,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $this->state->setAreaCode(Area::AREA_ADMINHTML);
```

Comment:
Do we need to set area code in the command if it's also emulated in `UpdateAssetInGrid.php` ?

Reply:
I wheni try save asset without area code exception is shown
![DeepinScreenshot_select-area_20200116151447](https://user-images.githubusercontent.com/2481206/72528094-eb1a0680-3872-11ea-8601-33fb8bfb9b34.png)